web/web/api/v1/sns_read.py

```
@@ -0,0 +1,30 @@
+from flask import Flask, request, Blueprint
+import json
+
+sns_read_api_bp =  Blueprint('sns_read_api_bp', __name__)
+
+
+@sns_read_api_bp.route('/sns_read', methods = ['GET', 'POST'])
+def sns():
+    # AWS sends JSON with text/plain mimetype
+    try:
+        js = json.loads(request.data)
+    except:
+        pass
```

if the parsing throws an exception and you pass the handling, line 17 and 19 would definitely blow up if reached. Either return early with an error or let it crash.

---

Not sure if I'm handling this the way you want it, lmk if there needs to be a change - I'm just returning an error message now